modules/olmv1-about-disconnected.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * extensions/catalogs/disconnected-catalogs.adoc
+
+:_mod-docs-content-type: CONCEPT
+
+[id="olmv1-about-disconnected_{context}"]
+= About disconnected support and the oc-mirror plugin in {olmv1}
+
+{olmv1-first} supports disconnected environments starting in {product-title} 4.18. After using the oc-mirror plugin for the {oc-first} to mirror the images required for your cluster to a mirror registry in your fully or partially disconnected environments, {olmv1} can function properly in these environments by utilizing either of the following sets of resources, depending on which oc-mirror plugin version you are using:
+
+* `ImageContentSourcePolicy` resources (automatically generated by oc-mirror plugin v1) and `ClusterCatalog` resources (must be manually created after using oc-mirror plugin v1))
```

Comment:
I think we eschew parentheses. 

```suggestion
* `ImageContentSourcePolicy` resources, which are automatically generated by oc-mirror plugin v1, and `ClusterCatalog` resources, which must be manually created after using oc-mirror plugin v1.
```
If not it looks like you have a double ) at the end instead of a period. 
```suggestion
* `ImageContentSourcePolicy` resources (automatically generated by oc-mirror plugin v1) and `ClusterCatalog` resources (must be manually created after using oc-mirror plugin v1).
```